packages/purpose-process/src/utilities/errorMappers.ts

```diff
@@ -109,6 +109,24 @@ export const suspendPurposeVersionErrorMapper = (
     .with("notValidVersionState", () => HTTP_STATUS_BAD_REQUEST)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
 
+export const createPurposeVersionErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("unchangedDailyCalls", () => HTTP_STATUS_BAD_REQUEST)
+    .with(
+      "organizationIsNotTheConsumer",
+      "organizationNotAllowed",
+      () => HTTP_STATUS_FORBIDDEN
+    )
+    .with(
+      "purposeNotFound",
+      "agreementNotFound",
+      "descriptorNotFound",
```

Comment:
these should be evaluated only when creating a new purpose.
If they rise when creating only a new version it's ok to return 500
```suggestion
```